frontend_tests/node_tests/emoji.js

```diff
@@ -23,11 +23,11 @@ run_test("get_canonical_name", () => {
     let canonical_name = emoji.get_canonical_name("green_tick");
     assert.equal(canonical_name, "green_tick");
 
-    canonical_name = emoji.get_canonical_name("thumbs_up");
-    assert.equal(canonical_name, "+1");
+    canonical_name = emoji.get_canonical_name("thumbs_up_sign");
+    assert.equal(canonical_name, "thumbs_up_sign");
```

Comment:
The "emoji-datasource-google" is from https://github.com/iamcal/emoji-data, which is where we get the sprites from. [cldr-json](https://github.com/unicode-org/cldr-json) doesn't provide a sprite sheet, nor does it provide emoji code, and so we need to use iamcal's even with CLDR's naming scheme.

Reply:
> It should be an explicit, considered, choice of its own.

Additionally, there will be migration headache with this PR, because the existing messages in people's self-hosted instances are still in the form of the names in old emoji_names.py.

Reply:
Yeah. I haven't tried to think through the implications of migrating the names; I agree that's something we'll want to be thoughtful about.

Reply:
I tested the effect of emoji renames post install. Rendered html messages render the emoji correctly, because their CSS class use the emoji code, which is the same regardless of the naming scheme. Reactions with old names also show up because of the CSS class.